package.json

```diff
@@ -26,6 +26,7 @@
 	},
 	"dependencies": {
 		"browserslist": "^3.2.7",
+		"extract-sourcemap": "^1.0.0",
```

Comment:
Since we’re using it only in tests, isn’t it actually a devDependency?

Reply:
Nicely spotted! I'll fix this.